simulate_mining.py

```diff
@@ -296,23 +355,33 @@ def main():
         os.mkdir("logs")
 
     timestamp = datetime.datetime.now().strftime("%Y%m%d%H%M%S")
+    short_timestamp = datetime.datetime.now().strftime("%H%M%S")
 
     logger = configure_logger("mining", timestamp, options.logging)
 
     logger.info(f"Command line arguments: {options}")
 
     stakers = build_stakers(logger, options, timestamp)
-    # Everyone starts off with coin age equal to 1, otherwise no block will be mined in the first epoch
-    coin_age = {staker: 1 for staker in range(options.num_stakers)}
+    num_stakers = len(stakers)
+    # all stakers but whales start off with coin age equal to maximum age (total number of stakers)
```

Comment:
A "whale", in the way is pretended to be simulated, would be some new identity willing to stake a rather big amount of wit on a pre-established population of stakers whose stake distribution is determined by the `distribution` parameter, and after the PoS was activated.

Reply:
I am not convinced this can be achieved in a real-world system, but we'll cross that bridge when necessary. I'll just merge the PR now so we can work from the same code base. 😄